src/extensionApi.ts

```diff
@@ -70,7 +70,9 @@ export class CommandHandler {
         }
 
         const stateObj: Protocol.ServerState = this.serversData.serverStatus.get(serverId);
-        if (stateObj.state === ServerState.STARTED) {
+        if ((!forced && stateObj.state === ServerState.STARTED)
```

Comment:
why dont we allow force stopping if the server is started? what's the idea? Can't we also forcefully stop it when it's already started?

Reply:
The stopServer method was always called with the force flag set to true. (Until now the stop method worked this way so i left it like this, maybe we should change it, no idea why it went launched with this flag always set) btw, because the terminate and the stop button do almost the same stuff, it's just a matter of visibility. Terminate is displayed when stop is hidden (during starting/stopping phase). This "if" is there just to keep the behaviour consistent with the visibility. But it's useless for the result of the method. It would have been enough just to check the server state

Reply:
Agree, we might have to revisit why we're always force stop (not just for terminating). But that would be in a new issue.